src/components/Task.vue

```diff
@@ -163,23 +192,24 @@ export default defineComponent({
   border-radius: 100%;
   display: inline-block;
 }
-.tag {
-  border-radius: 100%;
-  background-color: #cccccc;
-  display: inline-block;
-  font-size: 0.5rem;
-  padding-left: 0.2rem;
-  padding-right: 0.2rem;
-  margin-right: 1px;
-}
 .footer {
   display: flex;
   flex-direction: row;
   margin-top: 1px;
 }
+.label {
+  font-size: 0.55rem;
+  font-weight: 700;
+  text-align: left;
+  margin-bottom: 0.4rem;
+  margin-top: 0.1rem;
+  margin-left: 0.1rem;
+}
 .date {
-  font-size: 0.5rem;
+  margin-right: 0.1rem;
+  font-size: 0.55rem;
+  font-weight: 700;
   flex-grow: 1;
-  text-align: right;
+  float: right;
```

Comment:
Be careful with floats on a class like this. You might need a clear after using this.